src/services/formatting/smartIndenter.ts

```diff
@@ -287,6 +294,33 @@ module ts.formatting {
             }
         }
 
+        function getLineIndentationWhenExpressionIsInMultiLine(node: Node, sourceFile: SourceFile, options: EditorOptions): number {
+            if (node.parent.kind === SyntaxKind.CallExpression) {
+                let parentExpression = (<CallExpression>node.parent).expression;
+                let startingExpression = getStartingExpression(<CallExpression>parentExpression);
+
+                if (parentExpression === startingExpression) {
+                    return Value.Unknown;
+                }
+
+                let parentExpressionEnd = sourceFile.getLineAndCharacterOfPosition(parentExpression.end);
+                let startingExpressionEnd = sourceFile.getLineAndCharacterOfPosition(startingExpression.end);
+
+                if (parentExpressionEnd.line === startingExpressionEnd.line) {
+                    return Value.Unknown;
+                }
+
+                return findColumnForFirstNonWhitespaceCharacterInLine(parentExpressionEnd, sourceFile, options);
+            }
+            return Value.Unknown;
+            
+            function getStartingExpression(expression: CallExpression) {

```

Comment:
argument type should really be a union of `PropertyAccessExpression | CallExpression | ElementAccessExpression`. Now it works by chance because all these types have similarly named property `expression`.